felt/src/lib.rs

```diff
@@ -397,7 +459,8 @@ impl AddAssign for Felt252 {
 
 impl<'a> AddAssign<&'a Felt252> for Felt252 {
     fn add_assign(&mut self, rhs: &Self) {
-        self.value += &rhs.value;
+        // TODO: optimize and move upstream
```

Comment:
I think they have a `+=` now. It's not optimized tho, it's pretty much the same implementation.

Reply:
They have it only for owned values (i.e. for `a += b`, where both are not references) 😢